index.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-u2Qo5c5wL2fZ5m0z/rqZjLmZ9+vHjE/AqY0rU/QdugnfeE0k8zO6rU2qzUITjtWc6i1Kc2nh/gPmKv5fK/xJwg==" crossorigin="anonymous" >
+
+  </head>
+  <body>
+    <div class="portfolio">
+        <header class="details">
+            <div>
+            <img src="images/Time Style.png" alt="time" class="top-left-image">
+            <img src ="images/Group 4.png" alt="battery" class="top-right-image">
+        </div>
+            <div class="name">
+                <span class="inside-name">Kelvin</span>
+
+            </div>
+            <img src="images/Union.png" alt="union" class="dropdown-menu">
+        </header>
+        <section class="section-container">
+        <div class="socials-container">
+            <a href="#" class="github">
+                <img src="images/github.png" alt="github">
+            </a>
+            <a href="#" class="linkedin">
+                <img src="images/Linkedin icon.png" alt="linkedin">
+            </a>
+            <a href="#" class="angel-list">
+                <img src="images/angel list.png" alt="angel-list">
+            </a>
+            <a href="#" class="twitter">
+                <img src="images/twitter.png" alt="Twitter">
+            </a>
+            <a href="#" class="medium">
+                <img src="images/medium.png" alt="Medium">
+            </a>
```

Comment:
- [ OPTIONAL ] I think your social media links should be a list of icons a use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href Like this
  ```html
   <ul>
        <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
        <!-- other links here -->
   </ul>
  ```